### PR TITLE
provider/softlayer: support multi-vm builds

### LINKED ISSUE #9744: Softlayer stack build fails when machine count is 2.
## Current Behavior
image

### PR DESCRIPTION
**Pull request by rjeczalik**
Closes #9744.

### CODE REVIEW

**Review by gokmen — APPROVED** [review]

File: go/src/koding/kites/kloud/provider/softlayer/stack.go [hunk 1]
```diff
@@ -117,19 +117,30 @@ func (s *Stack) injectBootstrap(b *Bootstrap, guest map[string]interface{}) erro
...
-	s.Builder.InterpolateField(guest, name, "user_data")
+	count := 1
+	if n, ok := guest["count"].(int); ok && n > 1 {
+		count = n
+	}
 
-	kiteKey, err := s.BuildKiteKey(name, s.Req.Username)
-	if err != nil {
-		return err
+	var labels []string
```
Comment (cihangir) [inline go/src/koding/kites/kloud/provider/softlayer/stack.go]: we have the very same logic & code in other providers, could we unify them somehow?

**Comment by cihangir** [comment]
@ppknap i am merging this for a deploy, please review it in your earlier convenience

File: go/src/koding/kites/kloud/provider/softlayer/stack.go (hunk 1 above)
Comment (rjeczalik) [inline go/src/koding/kites/kloud/provider/softlayer/stack.go]: naturally - there's a todo for this: https://github.com/koding/koding/issues/9127

Comment (cihangir) [inline go/src/koding/kites/kloud/provider/softlayer/stack.go]: 👍

**Review by ppknap — COMMENTED** [review]
@cihangir LGTM

**Pull request merged**